hass/configurations.js

```diff
@@ -162,7 +162,7 @@ module.exports = {
       brightness_state_topic: true,
       brightness_command_topic: true,
       on_command_type: 'first',
-      state_value_template: '{{ "OFF" if value_json.value == 0 else "ON" }}',
+      state_value_template: '{{ "off" if value_json.value == 0 else "on" }}',
```

Comment:
I think that this should be left to `ON` and `OFF` as our discussion. Just lights with `template` schema needs to use `on|off` lower cases

Reply:
fixed.. keeping this with `ON` and `OFF` and changed just the light dimmer to `on` and `off`